internal/handlers/telegram/handler_test.go

```diff
@@ -0,0 +1,40 @@
+package telegram
+
+import (
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestDocument(t *testing.T) {
+	correct := "Someone shared a file (Random File) on Telegram with caption: 'Random Caption'."
+	updateObj := &tgbotapi.Update{
+		Message: &tgbotapi.Message{
+			From: &tgbotapi.User{
+				FirstName: "Someone",
```

Comment:
So this works perfectly fine, but I would actually like to try to standardize on using `"test"` when we can as the `FirstName` or `UserName` of our test users. That way we don't need to think super hard about which test cases use which username.

This isn't documented anywhere, but I would like to strive for all of our tests to be laid out in the same fashion.

Reply:
You are right. Standardizing would make it easier in the future.